### PR TITLE
W to g guide

### PR DESCRIPTION
**Pull request by sjsrey**
A complete draft here.

Please see if any areas are missing, or suggest changes of what is in this.

### CODE REVIEW

File: docs/user-guide/graph/w_g_migration.ipynb
Comment (jGaboardi) [inline docs/user-guide/graph/w_g_migration.ipynb:1]: * text after cell 26: misspell - ~~gaurenteed~~ >> guaranteed
* text after cell 43: misspell  - ~~change~~ >> changed
* text after cell 45: typo - "... method and ~~to~~ a property..."
* text after cell 46: typo - "... by design, ~~the~~ the weights..."
* text after cell 56: confusing language - "... where the index is the index is the id..."
* text after cell 59: misspell - "... powerful indexing ~~affored~~ afforded by pandas..."

**Comment by knaaptime** [comment]
looks great! 

...also nice eye james, i missed all those

**Comment by jGaboardi** [comment]
> looks great!
> 
> ...also nice eye james, i missed all those

I am James Bot

**Comment by knaaptime** [comment]
whoa weird, my computer just said that out loud in captain planet's voice

**Review by knaaptime — APPROVED** [review]

**Review by jGaboardi — APPROVED** [review]

**Review by martinfleis — APPROVED** [review]

**Pull request merged**